docs/api/version3.md

```diff
@@ -1145,6 +1145,55 @@ Get module summary from the Token module
 https://service.lisk.com/api/v3/token/summary
 ```
 
+### Token Available IDs
```

Comment:
@vardan10 Thank you for updating the document. Could also please add an entry in the list above?

Reply:
Applied